polygon/bor/borabi/abi.go

```diff
@@ -51,3 +55,13 @@ func ValidatorSetContractABI() abi.ABI {
 func StateReceiverContractABI() abi.ABI {
 	return stateReceiverABI
 }
+
+var methodId []byte = stateReceiverABI.Methods["commitState"].ID
+
+func EventTime(encodedEvent rlp.RawValue) time.Time {
```

Comment:
this func is not used (duplicate with the one in the heimdall package) and can be removed